compose/ui/ui/src/jvmMain/kotlin/androidx/compose/ui/input/pointer/PointerEventTimeoutCancellationException.jvm.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.input.pointer
+
+import kotlinx.coroutines.CancellationException
+
+actual class PointerEventTimeoutCancellationException actual constructor(
```

Comment:
I don't see `expect` for it, did I miss something?

Reply:
Thanks! I didn't commit it. Added now.